analysis_rsp.tex

```diff
@@ -0,0 +1,50 @@
+
+\subsubsection{Portal}
+
+\textbf{P-101}: User interface cone search on the object catalog, review the results overplotted on the HIPS map.
+
+\textbf{P-102}: Astronomical Data Query Language (ADQL) cone search on the object catalog with magnitude constraints, create a color-magnitude diagram in the results view.
+
+\textbf{P-103}: Create exploratory color plots of different trans-Neptunian object (TNO) populations and perform statistical analyses to determine potential correlations. 
```

Comment:
Partly covered by P-102 (do search, make plots). But "perform statistical analyses to determine potential correlations" as in regression fits to data -- this is not currently possible to do in the Portal and AFAIK not planned. So I think this one maybe needs to be deleted.

Reply:
Yes confirmed this is fixed - resolved.